### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by kryvets-vadym**
- [DEMO LINK](https://kryvets-vadym.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://kryvets-vadym.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Review by taraselnik — COMMENTED** [review]
IMHO great ) i hope you'll enjoy my review

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,66 @@
...
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```
Comment (mykola-koval) [inline src/index.html:6]:
```suggestion
    <meta 
      name="viewport"
      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
    >
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,66 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="body">
+    <div class="product">
```
Comment (mykola-koval) [inline src/index.html:18]: i guess you can solve this task without using this div
checklist - 6

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,66 @@
...
+
+      <div class="product__card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="productImage"
+          class="product__image"
+        >
+
+        <div class="product__info">
+
```
Comment (mykola-koval) [inline src/index.html:28]:
```suggestion
```

File: src/styles/card.css [hunk 4]
```diff
@@ -0,0 +1,117 @@
+.product {
+  margin: 0;
```
Comment (mykola-koval) [inline src/styles/card.css:2]: checklist - 7

File: src/styles/card.css [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+  padding: 0;
+  box-sizing: border-box;
```
Comment (mykola-koval) [inline src/styles/card.css:4]: you can add this to reset.css

File: src/styles/card.css [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+  width: 160px;
+  height: 134px;
+
+  margin-top: 32px;
+  margin-bottom: 40px;
+}
+
+.product__info {
+  display: flex;
+  flex-direction: column;
```
Comment (mykola-koval) [inline src/styles/card.css:33]: really need flex here?

**Review by mykola-koval — CHANGES_REQUESTED** [review]

File: src/styles/card.css (hunk 6 above)
Comment (kryvets-vadym) [reply src/styles/card.css]: > really need flex here?

i don't need it here, i missed it

**Review by Olexandr6 — APPROVED** [review]

**Pull request closed without merging**